supplementary_style_guide/glossary_terms_conventions/general_conventions/e.adoc

```diff
@@ -2,7 +2,7 @@
 [discrete]
 [[ee]]
 ==== image:images/yes.png[yes] ee subsystem (noun)
-*Description*: In Red Hat JBoss Enterprise Application Platform, the "ee" subsystem is used to configure functionality in the Jakarta Enterprise Edition platform. In general text, write in lowercase as one word. Use "EE subsystem" when referring to the ee subsystem in titles and headings.
+*Description*: In Red Hat JBoss Enterprise Application Platform, the _ee subsystem_ is used to configure functionality in the Jakarta Enterprise Edition platform. In general text, write in lowercase as one word. Use "EE subsystem" when referring to the ee subsystem in titles and headings.
```

Comment:
In other entries, on the EAP subsystem name was italicized. I'm thinking those probably should include "subsystem" in italics like this one does. (Ex: batch-jberet) if you wouldn't mind sweeping through those in the a-f files?

Reply:
Fixed from entries a-f. :+1: